Vagrantfile

```diff
@@ -0,0 +1,78 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://vagrantcloud.com/search.
+  config.vm.box = "fedora/34-cloud-base"
+
+
```

Comment:
You can delete one empty line here and in the end of the file.

Reply:
@pazMenachem (and everyone else, of course) A word about fixing issues raised by reviewers (and modifying PRs in general):

The final PR that we merge should include a logical representation of the change that we wanted to introduce to the codebase, rather than a representation of the review process.
This PR included 1 commit at first (which makes perfect sense). The fact that we had to fix something about that commit, should not make the PR now include more than one commit.

We should add additional commits only if we decided to split the existing content into multiple commits or if we decided to add additional content to the PR and we think that content belongs on a different commit (not too common).

In this PR, we have a second commit which changes stuff from the first commit, and that only introduces noise to our git history. All in all, not very often we rewrite the history of our merged codebase, but we should definitely rewrite the history of our PRs before they get merged, so they are as clean as possible.

Technically speaking, the way to apply fixes to an existing PR is as follows:

1. Make your changes locally on your PR branch.
2. Add them with `git add file1 file2...`
3. Amend the existing commit with `git commit --amend` (you'll get an editor screen asking you to save the commit message again).
4. Push your changes to the remote PR branch. You'd have to use the `-f` or `--force` flag.

For this PR, you can instead squash the commits, as explained on the second challenge.